ValetTouchIDTest/ValetTouchIDTestViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  ValetTouchIDTestViewController.swift
+//  Valet
+//
+//  Created by Eric Muller on 4/20/16.
+//  Copyright © 2016 Square, Inc. All rights reserved.
+//
+
+import UIKit
+
+
+final class ValetTouchIDTestViewController : UIViewController
+{
+    // MARK: Properties
+
+    @IBOutlet var textView : UITextView?
+    let secureEnclaveValet : VALSecureEnclaveValet
+    let username = "CustomerPresentProof"
+
+
+    // MARK: Initializers
+
+    required init?(coder aDecoder: NSCoder)
```

Comment:
on Monday let's discuss if it's possible to make these initializers not fail-able (though this may be the best approach). Short of that, this is the way to go for sure.

Some recommended reading: http://www.cocoawithlove.com/blog/2016/01/25/partial-functions-part-one-avoidance.html